File: src/main/java/com/example/demo/controller/UserController.java
package com.example.demo.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;

@RestController
@RequestMapping("/user")
public class UserController {

    private static final String URL_PRE = "";

    @RequestMapping("/login")
    public Object login(String username,String password) throws Exception {

        HashMap<Object, Object> map = new HashMap<>();
        map.put("token","123");
        return map;
    }

}

File: game-center-view/src/main/java/club/dreamccc/gamecenter/view/controller/UserController.kt
package club.dreamccc.gamecenter.view.controller

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/user")
class UserController {


    private val URL_PRE = "123";

    @RequestMapping("/test")
    fun test(username: String?, password: String?): Any {

        return "test";
    }

}


